pkg/secrethub/secret_version.go

```diff
@@ -272,3 +276,58 @@ func (c *Client) decryptSecretVersions(encVersions ...*api.EncryptedSecretVersio
 
 	return versions, nil
 }
+
+// Iterator returns a new iterator that retrieves all accounts in the given namespace.
```

Comment:
Maybe add to the comment that `IncludeSensitiveData` should be set if you want to retrieve the secret data, to make this extra clear and prevent confusion when the sensitive data isn't set.

Reply:
and to the interface as well.

Reply:
Good point!